src/main/java/org/prebid/server/bidder/huaweiads/model/util/HuaweiAdsConstants.java

```diff
@@ -0,0 +1,38 @@
+package org.prebid.server.bidder.huaweiads.model.util;
+
+public interface HuaweiAdsConstants {
+
+    String API_VERSION = "3.4";
```

Comment:
Constants should be public static final

Reply:
Variables in an interface are already implicitly public static final